concepts/modularization/README.md

```diff
@@ -0,0 +1,75 @@
+# Modularization
+
+Kyma provides Kubernetes building blocks. It should be easy to pick only those that are needed for the job and it should be easy to add new blocks to extend Kyma features. With the growing number of components, it is not possible to always install them all anymore. 
+
+- [Independent releases of modules](#independent-releases-of-modules)
+- [Dependencies between components](#dependencies-between-components)
+- [Release channels](#release-channels)
+- [Versioning](#versioning)
+- [Manifest operator](#manifest-operator)
+
+# Independent releases of modules
+With the growing number of components, it is hard to deliver features and fixes quickly and efficiently. Changes in manifests require a new release of Kyma. Operators (reconcilers) are tightly coupled and must be released together. In most cases, new component releases don't involve any API changes and could be delivered in a few minutes. 
```

Comment:
Will we have some additional source that would list all currently supported components? 
Or will the customer be able to basically instruct our installer to install whatever he likes as long as the component implements the operators contract?

Reply:
Kyma operator will install anything that is available in the release channel assigned to the Kyma installation. This way user can balance between stability and availability of the new (sometimes experimental features). I can imagine that users can set up their own release channel with their own selection of components and it should also work (at least in the open-source world).

Reply:
Does it mean that each team can release a component when it passes unit tests and static checks for example? When do we test kyma integration as a whole?

Reply:
@kasiakepka I think we should have some kind of integration tests before deployment of new component version for all clusters on Stage and Prod landscape.